CICD/ExtensionMethods.cs

```diff
@@ -889,6 +889,27 @@ public static async Task<Milestone> CloseMilestone(this IMilestonesClient client
         return updatedMilestone;
     }
 
+    /// <summary>
+    /// Returns a value indicating whether or not the GitHub issue is a standard issue.
+    /// </summary>
+    /// <param name="issue">The GitHub issue to check.</param>
+    /// <returns><c>true</c> if a standard issue.</returns>
```

Comment:
Refactor to below, throughout the codebase.
```suggestion
    /// <returns><c>true</c> if the issue is standard.</returns>
```